internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/server/VeniceServerTest.java

```diff
@@ -205,10 +205,6 @@ public void testStartServerAndShutdownWithPartitionAssignmentVerification() {
 
       cluster.stopVeniceServer(server.getPort());
 
-      // Create new servers so partition assignment is removed for the offline participant
-      cluster.addVeniceServer(featureProperties, new Properties());
-      cluster.addVeniceServer(featureProperties, new Properties());
-
```

Comment:
These lines are needed, otherwise the partitions will still be assigned to the original server.